friendsofsymfony/user-bundle/2.1/post-install.txt

```diff
@@ -0,0 +1,34 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What\'s next? </>
+<bg=blue;fg=white>              </>
+
+Make sure the twig is installed and the twig engine is turned on
+in framework.yaml
+
+  * If not, install twig, then add the following to
+    config/packages/framework.yaml:
+
+framework:
+    # ...
+    templating:
+        engines: [\'twig\']
+
+<fg=yellow>For sending e mails:</>
+  1. Set MAILER_SENDER_ADDRESS and MAILER_SENDER_NAME in your .env file
+  2. Configure the mailer service in config/packages/fos_user.yaml
+
+<bg=blue;fg=white>                                        </>
+<bg=blue;fg=white> Follow this steps in the Documentation </>
+<bg=blue;fg=white>                                        </>
+
+See https://symfony.com/doc/master/bundles/FOSUserBundle/index.html
+
+  1. Step 3: Create your User class
+  2. Step 4: Configure your application\'s security.yml
```

Comment:
Could we also do this in the recipe?

Reply:
Hmm... I don't see any way to modify the application's `security.yaml` through the recipe. Do you've any idea to solve that?